Uplift/Controllers/HomeViewController+Extensions.swift

```diff
@@ -115,12 +122,16 @@ extension HomeViewController: UICollectionViewDelegate, UICollectionViewDelegate
 
     func collectionView(_ collectionView: UICollectionView, didHighlightItemAt indexPath: IndexPath) {
         guard let cell = collectionView.cellForItem(at: indexPath) else { return }
-        cell.zoomIn()
+        if !(cell is TodaysClassesEmptyCell) {
+            cell.zoomIn()
+        }
     }
 
     func collectionView(_ collectionView: UICollectionView, didUnhighlightItemAt indexPath: IndexPath) {
         guard let cell = collectionView.cellForItem(at: indexPath) else { return }
-        cell.zoomOut()
+        if !(cell is TodaysClassesEmptyCell) {
+            cell.zoomOut()
+        }
```

Comment:
might be good to leave comment about why we don't want to zoom in/out for empty cell